source/MaterialXShaderGen/SgNode.cpp

```diff
@@ -57,6 +57,8 @@ SgEdgeIterator SgOutput::traverseUpstream()
 }
 
 
+const SgNode SgNode::NONE("");
+
```

Comment:
Dummy node used when there is no SgNode instance used but we still need to send something into shader generator methods. I will clean this up later, probably by using SgNode pointers instead and allowing nullptr to be sent in when there is no SgNode instance.